src/applications/letters/sass/letters.scss

```diff
@@ -220,3 +220,7 @@ label + div {
 .bullet-disc {
   list-style-type: disc;
 }
+
+.address-section {
+  line-height: 2.5;
```

Comment:
Just going to "talk out" my thoughts around this line spacing - it looks huge on the page.  The figma for this section looks like it has smaller line spacing.  However, the example in the Figma showing the AE Design Patterns version does also have quite a large line height.  So I think if we are going off of that, this should be fine?  I sort of want to tag in @sh0wn and get his thoughts on the line height on the screenshot (if you haven't run it by him already).